### DISCUSSION TITLE
crop with coordinates did't work well

### DISCUSSION THREAD

**Discussion by HinsChueng** [jsvine/pdfplumber#969] [Ask for help with specific PDFs]
**Description** :
Now I'm using the page.crop function to extract text I need with coordinates. 
There are 3 problems bothering me：
1. In this pdf, I extract with correct coordinates in the second page but I got wrong text from the first page.
2. The image cropped from page with coordinates did't match text in it.
3.  page's width should be shorter than it's height obviously, but it's not true in this pdf.
<img width="416" alt="image" src="https://github.com/jsvine/pdfplumber/assets/29364572/f48afc8e-274a-46f4-8045-27439b711dce">

**Pdf File**: [第11期 关于Accessibility的探究.pdf](https://github.com/jsvine/pdfplumber/files/12334404/11.Accessibility.pdf)

**Test Code**:
```python
import pdfplumber

pdf_name = '第11期 关于Accessibility的探究.pdf'
pdf_path_ = f'test_files/{pdf_name}'
pdf = pdfplumber.open(pdf_path_)
chars = pdf.pages[0].chars
for c in chars:
    if c['x1'] > 600:
        print(c['text'])

# width>height
print(pdf.pages[0].width)
print(pdf.pages[0].height)

# crop from the second page but text comes from the fist page, and image didn't match the text.
box = pdf.pages[1].crop((75.79969024658203, 83.29334259033203, 286.6417236328125, 579.9869384765625))
box.to_image(resolution=600).show()
print(box.extract_text())
```

**Comment by cmdlineluser** [comment]
How odd. 

It doesn't seem to be specific to `.crop` - for some reason page 2's `.chars` are those from page 1.

```python
>>> pdf.pages[0].extract_text() == pdf.pages[1].extract_text()
True
```

**Update 1:** Trying `pdfminer` directly, it extracts the text from **all pages** no matter what number I specify.

```python
from pdfminer.high_level import extract_text

print(extract_text("Downloads/11.Accessibility.pdf", page_numbers=[0]))
```
```
...
定名发表看法，本刊将在术语栏目发表大家的讨论意见。cccf@ict.ac.cn

80

81

 第 5 卷  第 11 期  2009 年 11 月                   第 5 卷  第 11 期  2009 年 11 月                  
```

**Update 2:** Tried some more tools: `pdftotext`, `pypdf`, `pypdfium2` all seem to have the same issue. 

`pymupdf` extracts each page of text correctly.

**Reply by HinsChueng** [reply]
Thanks for your testing and advice! I'm learning about `pymupdf ` now.
